client/src/tokens/colors.js

```diff
@@ -137,8 +137,8 @@ const colors = {
       contrastText: 'white',
     },
     DEFAULT: {
-      hex: '#007D7E',
-      hsl: 'hsl(180 100% 25%)',
+      hex: '#17009b',
+      hsl: 'hsl(249deg 100% 30%)',
```

Comment:
We should not be changing the core colours as part of this fix, these should always stay as is.

Reply:
Ok. Let me revert the changes